jazz_environments/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "jazz_environments",
+    "version": "0.0.1",
+    "description": "CRUD APIs for Environments",
+    "main": "index.js",
+    "private": true,
+    "author": {
+        "name": "Jazz"
+    },
+    "scripts": {
+        "test": "mocha"
+    },
+    "dependencies": {
+        "json-query": "*",
+        "request": "*",
+        "guid": "0.0.12",
+        "deasync": "*",
```

Comment:
Remove this dependency if not being used.

Reply:
removed